docs/Home.md

```diff
@@ -0,0 +1,87 @@
+# Google Kubernetes Engine Plugin Wiki
+
+The [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) (GKE) Plugin allows you to publish deployments built within Jenkins to your Kubernetes clusters running within GKE.
+
+_Plugin Information_
+
+View Google Kubernetes Engine [on the plugin site](https://plugins.jenkins.io/google-kubernetes-engine) for more information.
```

Comment:
The plugin page is mostly a mirror of the old wiki. Since you'll be changing it to link to this new documentation, it might make sense to remove this sentence entirely.